src/firefly/js/charts/dataTypes/SpectrumUnitConversion.js

```diff
@@ -176,3 +176,16 @@ const UnitXref = {
     }
 
 };
+
+/**
+ * returns X axis label combining spectral frame and redshift information
+ * @param {string} cname         the name of the column being evaluated
+ * @param {string} unit          the unit to get the info for
+ * @param {string} sfLabel       the label of spectral frame selected
+ * @param {string} redshiftLabel the label of redshift if rest frame, optional
+ * @returns {string}
+ */
+export const getCombinedXLabel = (cname, unit, sfLabel, redshiftLabel='') => {
+    const unitLabel = getUnitInfo(unit, cname).label;
```

Comment:
I understand why you named it such.  But, it makes more sense to just call it getXLabel, or getDefaultXLabel.  In the function, you can comment on what was taken into consideration when creating the label, which you already have.  There may be more in the future.

Reply:
Agreed, done!